content-src/components/DiscoveryStreamComponents/DSContextFooter/_DSContextFooter.scss

```diff
@@ -0,0 +1,111 @@
+$status-green: #058B00;
+$status-dark-green: #7C6;
+
+.story-footer {
+  color: var(--newtab-text-secondary-color);
+  display: flex;
+  height: 24px;
+  inset-inline-start: 0;
+  margin-top: 12px;
+  align-items: center;
+  transition: transform 0.3s;
+  transform-style: preserve-3d;
+
+  .story-sponsored-label,
+  .status-message {
+    @include dark-theme-only {
+      color: $grey-40;
+    }
+
+    font-size: 13px;
+    line-height: 24px;
+    color: $grey-50;
+  }
+
+  .status-message {
+    display: flex;
+    align-items: center;
+    height: 24px;
+
+    .story-badge-icon {
+      @include dark-theme-only {
+        fill: $grey-40;
+      }
+
+      fill: $grey-50;
+      height: 16px;
+      margin-inline-end: 6px;
+
+      &.icon-bookmark-removed {
+        background-image: url('../data/content/assets/icon-removed-bookmark.svg');
+      }
+    }
+
+    .story-context-label {
+      @include dark-theme-only {
+        color: $grey-40;
+      }
+
+      color: $grey-50;
+      flex-grow: 1;
+      font-size: 13px;
+      line-height: 24px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  }
+}
+
+.story-animate-enter {
+  opacity: 0;
+}
+
+.story-animate-enter-active {
+  opacity: 1;
+  transition: opacity 150ms cubic-bezier(0.07, 0.95, 0, 1) 300ms;
```

Comment:
Let's use the ease-in timing function here, the cubic bezier is too harsh.

Reply:
Ease-in looks more noticeable here (also switched to recording in Quicktime)
https://drive.google.com/open?id=1ZMBgZJIGvR9e-D_sTJ_lAjkG2YnTuW7V

Reply:
Looking good!